collection/Morgan; Morgan's Arsenal of DM Tools.json

```diff
@@ -0,0 +1,3905 @@
+
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "MADMT",
+				"abbreviation": "MADMT",
+				"full": "Morgan's Arsenal of DM Tools 1.1",
+				"authors": [
+					"Morgan"
+				],
+				"convertedBy": [
+					"Deri"
+				],
+                "version": "1.0.0",
+				"color": "771717",
+				"url": "",
+				"targetSchema": "1.0"
+			}
+		],
+        "dateAdded": 1707257718,
+        "dateLastModified": 1707257718
+	},
+
+
+	"book": [
+		{
+			"name": "Morgan's Arsenal of DM Tools 1.1",
+			"id": "MADMT",
+			"source": "MADMT",
+			"cover": {
+				"type": "external",
+				"url": ""
+			},

```

Comment:
```suggestion
```

Reply:
Hi,

What's the issue here? For my own, future reference, I'd rather not be submitting flawed files.

Thanks